components/Skeleton/__snapshots__/Skeleton.unit.test.jsx.snap

```diff
@@ -90,15 +94,15 @@ exports[`Skeleton component Should match the snapshot of a text skeleton 1`] = `
   animation: hDWTUO 1.5s ease-in-out infinite;
   display: inline-block;
   box-sizing: border-box;
-  height: 14px;
   width: 100%;
+  height: 14px;
 }
 
 .c1 {
   border-radius: 4px;
 }
 
-<div
+<span
```

Comment:
Why does only the Skeleton.Text snapshot render a span instead of a div? I didn't see changes in the unit tests that would justify this behavior, do you know what it could be?

Reply:
This pull request was created to address an issue (#437) related to semantic problems in the HTML structure of the Skeleton component. 

The suggestion in the issue was to use a `span` as the container element for `Skeleton.Text.` Therefore, in this pull request, I made the change to use a `span` as the **default element for Skeleton.Text**, as it was recommended to fix the semantic issue. 

There might not be any changes in the unit tests, as this is a structural change rather than a functionality change

Reply:
Sorry I caused confusion on this one. I understand the purpose of the tweak, but what I didn't understand was why there is only one snapshot that renders a span instead of a div. I saw now that the Skeleton.Text defined in line 41 of the components/Skeleton/Skeleton.jsx file contained an "as" attribute defined with the value "span". Ignore this question, please.